[2025-02-01T02:04:23.334+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_to_bigquery_dag.upload_to_bigquery manual__2025-02-01T02:04:16.383689+00:00 [queued]>
[2025-02-01T02:04:23.340+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_to_bigquery_dag.upload_to_bigquery manual__2025-02-01T02:04:16.383689+00:00 [queued]>
[2025-02-01T02:04:23.340+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-02-01T02:04:23.353+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): upload_to_bigquery> on 2025-02-01 02:04:16.383689+00:00
[2025-02-01T02:04:23.357+0000] {standard_task_runner.py:57} INFO - Started process 1459 to run task
[2025-02-01T02:04:23.360+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'gcs_to_bigquery_dag', 'upload_to_bigquery', 'manual__2025-02-01T02:04:16.383689+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/json_to_bigquery.py', '--cfg-path', '/tmp/tmp0e3s1y4b']
[2025-02-01T02:04:23.363+0000] {standard_task_runner.py:85} INFO - Job 116: Subtask upload_to_bigquery
[2025-02-01T02:04:23.393+0000] {task_command.py:410} INFO - Running <TaskInstance: gcs_to_bigquery_dag.upload_to_bigquery manual__2025-02-01T02:04:16.383689+00:00 [running]> on host 3f33197bb456
[2025-02-01T02:04:23.441+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_to_bigquery_dag' AIRFLOW_CTX_TASK_ID='upload_to_bigquery' AIRFLOW_CTX_EXECUTION_DATE='2025-02-01T02:04:16.383689+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-01T02:04:16.383689+00:00'
[2025-02-01T02:04:23.446+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-02-01T02:04:23.465+0000] {logging_mixin.py:150} INFO - üî¥ No data to insert into BigQuery. Î≥ÄÌôòÎêú Îç∞Ïù¥ÌÑ∞Í∞Ä ÏóÜÏäµÎãàÎã§!
[2025-02-01T02:04:23.465+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-01T02:04:23.472+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=gcs_to_bigquery_dag, task_id=upload_to_bigquery, execution_date=20250201T020416, start_date=20250201T020423, end_date=20250201T020423
[2025-02-01T02:04:23.503+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-02-01T02:04:23.517+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
